plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
```

Comment:
I like that this was made consistent with positioning relative to its callback function with the one earlier in this file, but what's the pattern we should follow with regards to being above/below the function declaration? At least in twenty* themes and built-in plugins (`hello.php`), it's placed below the function.